### PR TITLE
Circle CI: Configure Rubocop

### PR DESCRIPTION
**Pull request by ldlsegovia**
Configure review dog to run Rubocop with circle ci

image

### CODE REVIEW

File: app/inputs/ajax_filter_input.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 class AjaxFilterInput < SearchSelectFilterInput
   def to_html
     warn "[DEPRECATION] use :search_select_filter instead of :ajax_filter"
+
+
```
Comment (monkeyci) [inline app/inputs/ajax_filter_input.rb:5]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: app/inputs/ajax_filter_input.rb [hunk 2]
```diff
@@ -1,6 +1,11 @@
...
+
```
Comment (monkeyci) [inline app/inputs/ajax_filter_input.rb:6]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: app/inputs/ajax_filter_input.rb [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+
```
Comment (monkeyci) [inline app/inputs/ajax_filter_input.rb:7]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: app/inputs/ajax_filter_input.rb [hunk 4]
```diff
@@ -1,6 +1,11 @@
...
+
```
Comment (monkeyci) [inline app/inputs/ajax_filter_input.rb:8]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: app/inputs/color_picker_input.rb [hunk 5]
```diff
@@ -32,6 +32,6 @@ def render_custom_input
   end
 
   def load_control_attributes
-    load_data_attr(:palette, default: DEFAULT_PALETTE, formatter: :to_json)
+    load_data_attr(:palette, { default: DEFAULT_PALETTE, formatter: :to_json })
```
Comment (monkeyci) [inline app/inputs/color_picker_input.rb:35]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Redundant curly braces around a hash parameter.

**Review by rjherrera — APPROVED** [review]

**Pull request merged**